build.rs

```diff
@@ -86,6 +95,7 @@ fn start_test_module(out: &mut File, testsuite: &str) -> io::Result<()> {
             .expect("testsuite filename should be representable as a string")
             .replace("-", "_"),
     )?;
+    writeln!(out, "        #[cfg(test)]")?;
```

Comment:
Don't you want this above the `mod`?

Reply:
The rest of the contents of the `mod` were under #[test]` already, so this worked, however you're right that it'd be simpler this way. I've now updated the patch.